examples/data-fetch/pages/index.js

```diff
@@ -2,21 +2,25 @@ import React from 'react'
 import Link from 'next/link'
 import fetch from 'isomorphic-unfetch'
```

Comment:
Fetch does not need to be isomorphic any more, if I understand things correctly.

Reply:
Sure, `node-fetch` would be good here, in this case it would only be used server side.

Reply:
OK I'll change